saphanabootstrap-formula.spec

```diff
@@ -103,9 +105,9 @@ fi
 %{fdir}/states/%{fname}/%{ftemplates}
 %{fdir}/metadata/%{fname}
 
-%dir %attr(0750, root, salt) %{fdir}
```

Comment:
@arbulu89 I think it will conflict with the `salt-standalone-formula-configuration` package as it creates the directory with different permission. We need to set specific attribute values for the files that will be different from the directories permissions and keep these directory permissions. AFAIK chaging the default attribute to `%defattr(-,root,root)` should do the trick...